staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -116,6 +124,7 @@ func InstallPathHandler(mux mux, path string, checks ...HealthzChecker) {
 	klog.V(5).Info("Installing healthz checkers:", formatQuoted(checkerNames(checks...)...))
 
 	mux.Handle(path, handleRootHealthz(checks...))
+	// TODO: let's remove this in a future version
```

Comment:
It'd be good to (in a followup) state the reasoning--TODOs are sometimes blindly followed.

Reply:
Sure thing.